content/docs/core.mdx

```diff
@@ -81,15 +81,15 @@ Pomerium also provides [Docker container images](https://www.docker.com/resource
 - `:vX.Y.Z` corresponds to a [specific tagged release](https://github.com/pomerium/pomerium/tags).
 
   ```shell-session
-  $ docker run pomerium/pomerium:v0.25.0 --version
-  pomerium: 0.25.0-1704902203+e6ed4d53
+  $ docker run pomerium/pomerium:v0.27.0 --version
+  pomerium: 0.27.0-1704902203+e6ed4d53
   envoy: 1.28.0+eb930e32ab5555643e09d11d490e392d0a790c5a80eb0b0ebacb1046bdbb114d
```

Comment:
this is not an output from a new version

Reply:
Good catch, looks like the actual output should be this:

```
pomerium: 0.27.0+3dadcf18
envoy: 1.31.0+d879aa604b9bdfcb2ed669098a7d062862dc5ea1e90e540338b2e532554bb0a6
```

Reply:
Thanks for testing that. I'll update the output example.